capture/project_library/project.py

```diff
@@ -120,13 +128,28 @@ def get_plaintexts(self, start: Optional[int] = None, end: Optional[int] = None)
         """ Get plaintexts[start, end] from project.
         """
         if self.project_cfg.type == "cw":
-            if start and end:
+            if (start is not None) and (end is not None):
                 return self.project.textins[start:end]
+            elif start is not None:
+                return self.project.textins[start]
             else:
                 return self.project.textins
         elif self.project_cfg.type == "ot_trace_library":
             return self.project.get_plaintexts(start, end)
 
+    def get_ciphertexts(self, start: Optional[int] = None, end: Optional[int] = None):
+        """ Get ciphertexts[start, end] from project.
+        """
+        if self.project_cfg.type == "cw":
+            if (start is not None) and (end is not None):
+                return self.project.textouts[start:end]
+            elif start is not None:
+                return self.project.textouts[start]
+            else:
+                return self.project.textouts
+        elif self.project_cfg.type == "ot_trace_library":
+            return self.project.get_ciphertexts(start, end)
+
```

Comment:
Is this new function related to fixing array slicing? Or is it maybe in the wrong commit?

Reply:
You are right, it is related to this PR. So I've keep this here and moved the array slicing bug to a separate PR.